build/4-nfts-api.md

```diff
@@ -316,9 +316,13 @@ Collection can be created with a few features/functionalities:
 2 types of collections are supported:
 
 1. Generic collection (based on [OpenZeppelins ERC-721](https://docs.openzeppelin.com/contracts/3.x/erc721) NFT standard)
```

Comment:
Here we need to also explain that generic means erc721 on EVM (with additional features) and on substrate in means psp34.